src/scripts/project.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable import/no-cycle, import/no-named-default */
+import { renderProjects, renderTask, projectId } from './render';
```

Comment:
avoid circular dependencies this is usually a pointer to bad software design, see link https://stackoverflow.com/questions/51094117/dependency-cycle-detected-import-no-cycle